filebeat/harvester/config.go

```diff
@@ -6,42 +6,42 @@ import (
 	"time"
 
 	cfg "github.com/elastic/beats/filebeat/config"
-	"github.com/elastic/beats/filebeat/input"
+	"github.com/elastic/beats/filebeat/harvester/processor"
 	"github.com/elastic/beats/libbeat/common"
 )
 
 var (
 	defaultConfig = harvesterConfig{
-		BufferSize:      cfg.DefaultHarvesterBufferSize,
-		DocumentType:    cfg.DefaultDocumentType,
+		BufferSize:      16 << 10, // 16384
```

Comment:
it's easier to get the number wrong when using `16384` over `16 * units.Kibi` for example.